spec/rollbar_spec.rb

```diff
@@ -190,6 +190,29 @@
         notifier.log('error', exception, extra_data, 'exception description')
       end
 
+      context 'when the extra data is something that supports to_h' do
+        # This is faking ActiveModel::Errors, which is the use case this feature was
+        # originally added in order to support.
+        # Generated from a prod-app by doing the following, where `opn` is some Rails/ActiveRecord model
+        # irb(main):007:0> opn.errors
+        # => #<ActiveModel::Errors:0x000055b2d9bab440 @base=#<FooBarModel id: 6031, ..., @messages={:payload=>["can't be blank"]}>
+        # irb(main):008:0> opn.errors.to_h
+        # => {:payload=>"can't be blank"}
+        FakeActiveModelErrors = Struct.new(:messages) do
+          def to_h
+            messages.to_h[:messages]
+          end
+        end
+
+        let(:extra_data) { FakeActiveModelErrors.new(messages: { "payload": "can't be blank" }) }
+        let(:extra_data_hash) { { "payload": "can't be blank" } }
```

Comment:
Hashrocket syntax is required in this project for back-compatibility, and single quotes for plain (non-interpolated) strings.
e.g.
`{ :payload =>  'can't be blank' }`